redaxo/src/core/lib/login/backend_login.php

```diff
@@ -197,6 +197,9 @@ public function changedPassword(
 
         if (null !== $passwordHash) {
             parent::changedPassword($passwordHash);
+            if (null !== $this->getUser()) {
+                rex_user_session::getInstance()->removeSessionsExceptCurrent($this->getUser()->getId());
```

Comment:
```suggestion
            if (null !== $user = $this->getUser()) {
                rex_user_session::getInstance()->removeSessionsExceptCurrent($user->getId());
```

Um die psalm meldung zu verhindern..